ide/projectuiapi.base/src/org/netbeans/spi/project/ui/ProjectProblemsProvider.java

```diff
@@ -187,7 +187,7 @@ public static Result create(
         private final String description;
         private final ProjectProblemResolver resolver;
 
-        private ProjectProblem(
+        public ProjectProblem(
```

Comment:
The constructor is now part of the API. Needed? If so, add Javadoc, `@since` tag, etc.

Reply:
Good catch ! It's a leftover prototyping, actually the factory methods are sufficient. Will remove.

Reply:
Fixed in 39596e2d6f